test/providers.ts

```diff
@@ -7,6 +7,7 @@ export const images = [
     fastly: { url: '/test.png' },
     glide: { url: '/test.png' },
     imgix: { url: '/test.png' },
+    unsplash: { url: '/unsplash/photo-1606112219348-204d7d8b94ee' },
```

Comment:
Why we need `/unsplash`? Correct URL seems like [this](https://images.unsplash.com/photo-1606112219348-204d7d8b94ee?w=200) (notice the one I used in current examples was for showcasing alias)

Reply:
Thanks I was a little confused by the alias.

```suggestion
    unsplash: { url: '/photo-1606112219348-204d7d8b94ee' },
```